src/components/include/utils/make_shared.h

```diff
@@ -34,6 +34,7 @@
 #define SRC_COMPONENTS_INCLUDE_UTILS_MAKE_SHARED_H_
 
 #include <new>
+#include <stddef.h>
```

Comment:
@AByzhynar it was my suggestion. Because excatly in this header or also in `cstddef` the definition for the `NULL` is located. And on Victoria's machine we had compilation error